moveit_core/profiler/src/profiler.cpp

```diff
@@ -175,7 +174,7 @@ void Profiler::console()
   std::stringstream ss;
   ss << std::endl;
   status(ss, true);
-  ROS_INFO_STREAM_NAMED("profiler", ss.str());
+  ROS_INFO_STREAM_NAMED("profiler", ss.str().c_str());
```

Comment:
Shouldn't this use `RCLCPP_INFO`? Also, why is c_str() necessary here?

Reply:
@henningkayser I believe this was explained in a previous discussion. If you remove that you'll find the following:

```bash
/Users/victor/ros2_moveit_ws/src/moveit2/moveit_core/profiler/src/profiler.cpp:177:37: error: cannot pass object of non-trivial type 'std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >::string_type' (aka 'basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >') through variadic function; call will abort at runtime [-Wnon-pod-varargs]
```
The new macros impose certain limitations.

Reply:
I see, I still don't get why you redefine ROS_INFO_STREAM_NAMED and don't use `RCUTILS_LOG_INFO` directly.

Reply:
You are right @henningkayser, thanks https://github.com/ros-planning/moveit2/pull/9/commits/a3e6d27e14671a64a6bb7f8518207124be967625